libraries/AP_NavEKF3/AP_NavEKF3_core.cpp

```diff
@@ -160,14 +160,16 @@ bool NavEKF3_core::setup_core(uint8_t _imu_index, uint8_t _core_index)
     }
 #endif
 
-    GCS_SEND_TEXT(MAV_SEVERITY_INFO, "EKF3 IMU%u buffs IMU=%u OBS=%u OF=%u EN:%u dt=%.4f",
+//removed GCS spam for a user unintelligible message, but left code in case debugging is required in the future
+
+    /* GCS_SEND_TEXT(MAV_SEVERITY_INFO, "EKF3 IMU%u buffs IMU=%u OBS=%u OF=%u EN:%u dt=%.4f",
```

Comment:
```suggestion
    /*
    removed GCS spam for a user unintelligible message, but left code in case debugging is required in the future
    GCS_SEND_TEXT(MAV_SEVERITY_INFO, "EKF3 IMU%u buffs IMU=%u OBS=%u OF=%u EN:%u dt=%.4f",
```

Reply:
done